main/modules/helpers.js

```diff
@@ -33,7 +33,7 @@ function takeScreenshot() {
     } else {
       const filters = [{ name: 'PNG file', extensions: ['png'] }]
       const fileName = dialog.showSaveDialogSync(win, { filters })
-      if (fileName) fs.writeFile(fileName + '.png', image.toPNG(), () => {})
```

Comment:
Are you sure about this change?  I tested this by changing the `if` to always pick this branch and found that it needs the '.png' extension to be added.

Reply:
@Quiescent  I tested this on a MBA big sur and ubuntu focal on `prod` branch and it still worked for me

Reply:
I'm talking about this when I refer to the "prod" branch of the if statement.
You can test this logic by setting the condition to `false` in the if statement.